compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/BallerinaLexer.java

```diff
@@ -1719,6 +1750,7 @@ private boolean processDocumentationReference(int nextChar) {
             case LexerTerminals.MODULE:
             case LexerTerminals.FUNCTION:
             case LexerTerminals.PARAMETER:
+            case LexerTerminals.CONST:
```

Comment:
Support for the special keyword `const` was added, as it is going to be included in the spec. (https://github.com/ballerina-platform/ballerina-spec/issues/336)
Keyword is already supported in the DocumentationAnalyzer and used in the unit tests as well.